### DISCUSSION TITLE
Put your bug found using ChatGPT here

### DISCUSSION THREAD

**Discussion by stan6** [SOEN345-WINTER2024/discussion#19] [Announcements]
Refer to https://github.com/orgs/SOEN345-WINTER2024/discussions/3 for example bug report

**Comment by ChemsCode** [comment]
## Time Zone Search Results Flicker Unexpectedly

**Description:**
I have encountered a bug within the "Time Zones" functionality of the Unitto app, where the search results for time zones flicker, disappearing and reappearing suddenly for a few seconds. This issue was found using the ChatGPT-Assisted UI Testing method.

**Steps to Reproduce:**
1. Launch the Unitto App.
2. Select the Hamburger Menu at the top left corner.
3. Navigate to the "Time Zones" functionality.
4. Tap on the "+" button to add a new timezone.
5. Enter any city or timezone in the search field.
6. Observe the search results flickering, appearing and disappearing randomly for a few seconds.

**Expected Behavior:**
The search results should remain stable on the screen, allowing the user to select their desired time zone without any visual disturbances.

![image](https://github.com/SOEN345-WINTER2024/mp2-ChemsCode/assets/90106593/c88946a2-7baf-48b3-bbe8-bef79260071f)


**Actual Behavior:**
The search results flicker, making it challenging to select a time zone due to the results disappearing and reappearing randomly.

![bug-report-345-unitto](https://github.com/SOEN345-WINTER2024/mp2-ChemsCode/assets/90106593/0ee4a6d5-1219-4b89-b7e2-aca62d710a56)


**Device Information:**
- Device Model: Google Pixel 3a
- Android Version: Android 14.00

[Link to Issue](https://github.com/sadellie/unitto/issues)

This issue significantly impacts the usability of the time zone functionality, and addressing it would enhance the overall user experience within the Unitto app.

**Reply by stan6** [reply]
Hi, @ChemsCode thank you for sharing this bug. I realized that the link to issue only link to the GitHub issue page of the app. Is it because the repository has been recently marked as archived and you can longer submit a GitHub issue (if this is the case, then it is fine)?

**Reply by ChemsCode** [reply]
Hi @stan6, yes unfortunately I wasn't able to publish an issue as the repository recently got archived by the owner.

**Comment by jesseythach** [comment]
# Images-to-PDF App Bug Report:

## How to reproduce:
1. Launch the app.
2. On the top left, tap on the hamburger menu.
3. Scroll down to find the "Settings".
4. Tap on the "Theme default value: System" option to change the app's theme.
5. Select any theme.

## Actual Behavior
After changing the theme, the user is directed to the main screen and it shows that the user is in the "Settings" page from the navigation menu by clicking on the hamburger menu.
<img src="https://github.com/SOEN345-WINTER2024/discussion/assets/94651491/c81efbe5-d997-4884-903e-82f9a03db1c3" width="500">
<img src="https://github.com/SOEN345-WINTER2024/discussion/assets/94651491/0bea8d53-9b4a-4f56-9805-bac0c4c0cec3" width="500">

## Expected Behavior
The user is expected to be on the Settings page if the highlight option in the navigation menu is the "Settings" option.
<img src="https://github.com/SOEN345-WINTER2024/discussion/assets/94651491/2e40f27c-a0e7-43e2-9e16-5c28474609c0" width="500">
<img src="https://github.com/SOEN345-WINTER2024/discussion/assets/94651491/1be2f0f4-be45-45a0-b813-2a15cad1931a" width="500">

App version: 9.0.1
Android version: 14

This issue is reported in this bug report: https://github.com/Swati4star/Images-to-PDF/issues/1163

**Comment by Mika24boss** [comment]
### How to reproduce
1. Launch the app
2. Long tap on the map to select a location
3. Swipe up to reveal options and tap on Nearby Stations
4. Pan the map until you see station markers
5. Zoom in and out to see their colours alternate between the normal red/white and black

(The same problem happens in dark theme)

### Actual behaviour
![bug](https://github.com/SOEN345-WINTER2024/discussion/assets/77291451/76a99eb0-479e-4c91-8c6b-84f1176d0f64)

### Expected behaviour
Zooming in and out shouldn't affect the icons.

### Device information:
App version: 2.2.1
Android version: 14.0
Pixel 7

This issue is reported to [Station icons' colours become black when zooming in and out #941](https://github.com/grote/Transportr/issues/941)

**Comment by LachezaraLaz** [comment]
**Describe the bug**
When trying to encrypt a file, a popup appears with the possibility to write a password but the popup doesn't scroll and does not let the keyboard disappear to see the whole page. Makes certain buttons on that page inaccessible.

**To Reproduce**
Steps to reproduce the behavior:
Go to a file
Click on the 3 vertical points of said file
Click on "Encrypt" button
Try to scroll to access buttons (e.g Cancel)
See error - buttons inaccessible

**Expected behavior**
Either the keyboard should be able to disappear when swipe down or the popup should be able to scroll up.

**Screenshots**
Not applicable

**Smartphone (please complete the following information):**
Device: Galaxy S7 Edge
OS: Android 8.0.0
Rooted: No
Version: Android 8.0.0

**Additional context**
Add any other context about the problem here.

This issue is reported to https://github.com/TeamAmaze/AmazeFileManager/issues/4141 

**Found bug using UI Testing**

**Comment by CarlaShawi** [comment]
## [Simple Gallery App](https://github.com/SimpleMobileTools/Simple-Gallery)

**Issue Title:** "New Folder Not Found in Search Within Simple Gallery App"

**Description:**

In the Simple Gallery app, after creating a new folder and adding pictures to it, the folder successfully appears on the view folders page. However, attempting to search for this newly created folder results in an error message indicating no items found, with a suggestion to switch to file search across all available folders.

This issue suggests a potential discrepancy in how the Simple Gallery app indexes or recognizes folders for search operations, possibly omitting newly created folders from the search index until a certain refresh or update action is triggered outside the normal folder creation process. This impacts the user experience by hindering the ability to quickly access specific folders in real-time.

**Steps to Reproduce:**

1.  Open Simple Gallery App.
2. Click on the three dots to get more options menu.
3. Click on Create New Folder.
4. Name the new folder and click ok.
5. Add pictures to the newly created folder (optional).
6. Verify that the new folder displays on the view folders page.
7. Use the search bar to locate the newly created folder by its name.

**Actual Behavior:** It displays a message that says "No items found. With the suggestions to Switch to file search across all available folders."

[Screen_recording_20240404_174645.webm](https://github.com/SOEN345-WINTER2024/discussion/assets/91223318/e99497b8-f190-4d72-b8e8-a4ebde15527b)



**Expected Behavior:** The search function should locate and display the newly created folder to the user.

**Device Information:**

Device Model: Pixel 6 Pro API 34
OS Version: Android version 14
App Version: Simple Gallery 6.28.1

**Link to issue reported:** [issue](https://github.com/SimpleMobileTools/Simple-Gallery/issues/3067)

**Comment by eo2000** [comment]
# Muzei Wallpaper App

Description:
When uploading photos for the wallpaper, the wallpaper is reloaded with a black screen when the device is unlocked. 

How to reproduce:
1. Open Muzei App.
2. Click on sources.
3. Click on My Photos
4. Upload photos 
5. Click the tick to confirm 
6. Lock phone
7. Unlock phone

Actual behaviour: 
<img width="298" alt="Screenshot 2024-04-07 at 3 32 23 PM" src="https://github.com/SOEN345-WINTER2024/discussion/assets/116772744/d6a08b6b-8b10-46db-a47b-33762bbec680">

Expected behaviour: 
The wallpaper should be the photos selected, not a black screen.

Device Model: Pixel 6 API 27
Android Version: Android 8.1 
App version: v3.5.0 

Link to issue reported: [issue](https://github.com/muzei/muzei/issues/814)

**Reply by stan6** [reply]
@eo2000 hi, your link to issue is redirect to an Github issue that is completely different from what you have submitted here. The user submitting the issue is also different. Can you provide the correct link?

**Reply by eo2000** [reply]
@stan6 Hi, I uploaded the correct link! Thanks!

**Comment by DobDub** [comment]
Hassan Moharram - 40158285
Markor app: https://github.com/gsantner/markor
Bug: FileBrowser: File Modification Time not updated until app restart (https://github.com/gsantner/markor/issues/2196)
Steps to reproduce:
1. open a file already created
2 . edit the file
3. close the file (keep app open)
error: file modification time does not change
![MarkorError](https://github.com/SOEN345-WINTER2024/discussion/assets/51329227/b857a83b-fc8d-41fe-b069-8414ec4434f1)
The time does not show here but it should say 5:01PM
App version: 2.5.0
Android version: 14
Device: Galaxy s23 Ultra

**Reply by stan6** [reply]
@DobDub hi, your link to issue is redirect to an Github issue that is completely different from what you have submitted here. The user submitting the issue is also different. Can you provide the correct link where you have reported the bug?

**Reply by DobDub** [reply]
https://github.com/gsantner/markor/issues/2196
The error is when editing a note the time stamp does not update until the app restarts which is the same issue here
